internal/fs/inode/dir_test.go

```diff
@@ -1378,7 +1378,7 @@ func (t *DirTest) DeleteChildFile_TypeCaching() {
 func (t *DirTest) DeleteChildDir_DoesntExist() {
 	const name = "qux"
 
-	err := t.in.DeleteChildDir(t.ctx, name, false)
+	err := t.in.DeleteChildDir(t.ctx, name, false, t.in)
```

Comment:
is the last parameter correct?. t.in is the pareent and you need to pass child to DeleteChildDir()

Reply:
Thanks for catch. Fixed